kasa/protocols/smartprotocol.py

```diff
@@ -45,15 +46,28 @@
     "original_device_id": lambda x: "REDACTED_" + x[9::],  # Strip children
     "nickname": lambda x: "I01BU0tFRF9OQU1FIw==" if x else "",
     "mac": mask_mac,
-    "ssid": lambda x: "I01BU0tFRF9TU0lEIw=" if x else "",
+    "ssid": lambda x: "I01BU0tFRF9TU0lEIw==" if x else "",
```

Comment:
`I01BU0tFRF9TU0lEIw=` is incorrect and will raise an error.

Reply:
Oh, nice catch. cyberchef and base64 command decoded it just fine, but python's b64decode did not.